Model/Api.php

```diff
@@ -11,28 +11,32 @@
 use Elgentos\PrismicIO\Api\ConfigurationInterface;
 use Elgentos\PrismicIO\Exception\ApiNotEnabledException;
 use Elgentos\PrismicIO\Model\Api\CacheProxy;
+use Exception;
 use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Store\Model\StoreManagerInterface;
 use Prismic\Api as PrismicApi;
+use Psr\Log\LoggerInterface;
 
 class Api
 {
+    private ConfigurationInterface $configuration;
 
-    /** @var ConfigurationInterface */
-    private $configuration;
-    /** @var StoreManagerInterface */
-    private $storeManager;
-    /** @var CacheProxy */
-    private $cacheProxy;
+    private StoreManagerInterface $storeManager;
+
+    private CacheProxy $cacheProxy;
+
+    private LoggerInterface $logger;
```

Comment:
You added the logger, but I don't see it being used anywhere. Is that correct?

Reply:
Remove